### ISSUE TITLE
Object extending a Class doesn't resolve instance variables declared in the class

### ISSUE THREAD

**Issue by javierfernandes** [uqbar-project/wollok#213]
Example:

```

object pepita extends Golondrina {
}

class Golondrina {
	var energia = 100
	method volar(kms) {
		this.modificar(-kms * 2)
	}
	method modificar(n) {
		energia += n 
	}
	method getEnergia() = energia
}
```

when you do
```
pepita.volar(2)
```
it fails with 

```
org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/test.wtest:[4]: test "testX" {

	assert.that(true)
	
	pepita.volar(2)
	
	assert.that(pepita.getEnergia() == 96)

}
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.eval(WollokInterpreterEvaluator.java:131)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:137)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:1)
	at org.eclipse.xtext.xbase.lib.IteratorExtensions.fold(IteratorExtensions.java:627)
	at org.eclipse.xtext.xbase.lib.IterableExtensions.fold(IterableExtensions.java:536)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.evalAll(WollokInterpreterEvaluator.java:140)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator._evaluate(WollokLauncherInterpreterEvaluator.java:62)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:159)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.interpret(WollokInterpreter.java:112)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.interpret(WollokInterpreter.java:99)
	at org.uqbar.project.wollok.launch.WollokLauncher.doSomething(WollokLauncher.java:39)
	at org.uqbar.project.wollok.launch.WollokChecker.doMain(WollokChecker.java:81)
	at org.uqbar.project.wollok.launch.WollokLauncher.main(WollokLauncher.java:29)
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/test.wtest:[8]: pepita.volar(2)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.eval(WollokInterpreterEvaluator.java:131)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:137)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:1)
	at org.eclipse.xtext.xbase.lib.IteratorExtensions.fold(IteratorExtensions.java:627)
	at org.eclipse.xtext.xbase.lib.IterableExtensions.fold(IterableExtensions.java:536)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.evalAll(WollokInterpreterEvaluator.java:140)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator._evaluate(WollokLauncherInterpreterEvaluator.java:76)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:163)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 13 more
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/objects.wlk:[7]: {
		this.modificar(-kms * 2)
	}
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable.eval(AbstractWollokCallable.java:126)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable$1.apply(AbstractWollokCallable.java:98)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.performOnStack(WollokInterpreter.java:155)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable.call(AbstractWollokCallable.java:110)
	at org.uqbar.project.wollok.interpreter.core.WollokObject.call(WollokObject.java:103)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._call(WollokInterpreterEvaluator.java:725)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.call(WollokInterpreterEvaluator.java:813)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.perform(WollokInterpreterEvaluator.java:721)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:691)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:157)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 22 more
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/objects.wlk:[8]: this.modificar(-kms * 2)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.eval(WollokInterpreterEvaluator.java:131)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:137)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:1)
	at org.eclipse.xtext.xbase.lib.IteratorExtensions.fold(IteratorExtensions.java:627)
	at org.eclipse.xtext.xbase.lib.IterableExtensions.fold(IterableExtensions.java:536)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.evalAll(WollokInterpreterEvaluator.java:140)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:596)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:117)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 33 more
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/objects.wlk:[10]: {
		energia += n 
	}
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable.eval(AbstractWollokCallable.java:126)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable$1.apply(AbstractWollokCallable.java:98)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.performOnStack(WollokInterpreter.java:155)
	at org.uqbar.project.wollok.interpreter.AbstractWollokCallable.call(AbstractWollokCallable.java:110)
	at org.uqbar.project.wollok.interpreter.core.WollokObject.call(WollokObject.java:103)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._call(WollokInterpreterEvaluator.java:725)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.call(WollokInterpreterEvaluator.java:813)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.perform(WollokInterpreterEvaluator.java:721)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:691)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:157)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 42 more
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/objects.wlk:[11]: energia += n
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.eval(WollokInterpreterEvaluator.java:131)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:137)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator$1.apply(WollokInterpreterEvaluator.java:1)
	at org.eclipse.xtext.xbase.lib.IteratorExtensions.fold(IteratorExtensions.java:627)
	at org.eclipse.xtext.xbase.lib.IterableExtensions.fold(IterableExtensions.java:536)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.evalAll(WollokInterpreterEvaluator.java:140)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:596)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:117)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 53 more
Caused by: org.uqbar.project.wollok.interpreter.WollokInterpreterException: Error evaluating line file:/Users/jfernandes/Downloads/wollok-macosx/wollok.app/Contents/MacOS/workspace/pruebaWollok/src/objects.wlk:[11]: energia
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:203)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.eval(WollokInterpreterEvaluator.java:131)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator.performOpAndUpdateRef(WollokInterpreterEvaluator.java:660)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:629)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:115)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 62 more
Caused by: org.uqbar.project.wollok.interpreter.UnresolvableReference: Cannot resolve reference energia
	at org.uqbar.project.wollok.interpreter.core.WollokNativeLobby.resolve(WollokNativeLobby.java:75)
	at org.uqbar.project.wollok.interpreter.core.WollokObject.resolve(WollokObject.java:223)
	at org.uqbar.project.wollok.interpreter.context.CompositeEvaluationContext.resolve(CompositeEvaluationContext.java:68)
	at org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator._evaluate(WollokInterpreterEvaluator.java:222)
	at org.uqbar.project.wollok.launch.WollokLauncherInterpreterEvaluator.evaluate(WollokLauncherInterpreterEvaluator.java:151)
	at org.uqbar.project.wollok.interpreter.WollokInterpreter.eval(WollokInterpreter.java:191)
	... 67 more

```

**Comment by javierfernandes** [comment]
Fixed in dev branch, milestone v1.2 Buddha

**Comment by javierfernandes** [comment]
<img width="272" alt="screen shot 2015-09-16 at 15 47 53" src="https://cloud.githubusercontent.com/assets/4428120/9934782/c8437926-5d29-11e5-87c9-d1e3b0a3757d.png">
 Sample image for the release notes
